### PR TITLE
CI build template patch

### PR DESCRIPTION
**Pull request by aminya**
This is a patch for #21230 

It includes the code signing keys in the code map and uses Azure macros for that instead. Azure does not pass secret variables to the program without using macros.

### CODE REVIEW

**Review by DeeDeeG — APPROVED** [review]
Looks good to me.

Note to maintainers: Pull Request CI runs always skip code signing. If you want to, you can test that code-signing truly works by pushing this to a branch starting with `electron-`. For example: `electron-test-code-signing`. That will trigger the production pipeline. And branch names starting with `electron-` are considered a "signed zip branch" name, so the Windows and macOS apps will be signed in the CI run.

**Review by sadick254 — APPROVED** [review]

**Comment by sadick254** [comment]
@DeeDeeG This one just slipped through the cracks for us. Thanks for the pointer.

File: script/vsts/platforms/templates/build.yml [hunk 1]
```diff
@@ -4,18 +4,12 @@ steps:
...
-        $ATOM_WIN_CODE_SIGNING_CERT_PASSWORD=$env:ATOM_WIN_CODE_SIGNING_CERT_PASSWORD
       }
-      if ($env:AGENT_OS -eq "Darwin") {
-        $ATOM_MAC_CODE_SIGNING_CERT_DOWNLOAD_URL=$env:ATOM_MAC_CODE_SIGNING_CERT_DOWNLOAD_URL
-        $ATOM_MAC_CODE_SIGNING_CERT_PASSWORD=$env:ATOM_MAC_CODE_SIGNING_CERT_PASSWORD
-        $ATOM_MAC_CODE_SIGNING_KEYCHAIN=$env:ATOM_MAC_CODE_SIGNING_KEYCHAIN
-        $ATOM_MAC_CODE_SIGNING_KEYCHAIN_PASSWORD=$env:ATOM_MAC_CODE_SIGNING_KEYCHAIN_PASSWORD
+      elseif ($env:AGENT_OS -eq "Darwin") {
         $AC_USER=$env:AC_USER
         $AC_PASSWORD=$env:AC_PASSWORD
```
Comment (DeeDeeG) [inline script/vsts/platforms/templates/build.yml:10]: Apologies for the back and forth.

These need to be moved to the `env:` block.

```yml
AC_USER: $(AC_USER)
AC_PASSWORD: $(AC_PASSWORD)
```

That allows for notarization to work on macOS.

Comment (aminya) [reply script/vsts/platforms/templates/build.yml:10]: Fixed in #21269

**Pull request merged**